src/routes/recipient/index.js

```diff
@@ -42,6 +43,11 @@ router.get(
   checkRecipientIdParam,
   transactionWrapper(getGoalsByIdandRecipient),
 );
+router.put(
+  '/:recipientId/markSimilarGoals',
```

Comment:
Let's avoid route paths that require case sensitivity if possible

Reply:
```suggestion
  '/:recipientId/mark-similar-goals',
```